src/docs/faq.md

```diff
@@ -1,88 +1,101 @@
+### My balance shows zero on coolstars portal, what should I do?
+* It may take up to 48 hours for balance to appear in her portal, though it is normally less then 2.
 
-#### How do I buy Windows audio drivers in bulk?
-* A discount is only available for Chromebooks containing an `Intel 6th gen processor (up to Core i5)`, `N3350`, `N4000`, or, `N4020` processor, otherwise you must pay full price. You must purchase 10 or more licenses to be eligible for a discount. If you are eligible for a discount, you should ask in the [Chrultrabook Forum](https://forum.chrultrabook.com).
+### How do I buy the Windows audio/Thunderbolt drivers?
+* Please see [CoolStar's driver portal](https://coolstar.org/chromebook/driverlicense/login.html) for a step by step tutorial on purchasing drivers.
 
-#### Can I use one driver license across multiple devices?
-* No. One license per device.
+### Can I get AVS or SOF drivers for free?
+* No. Only the Linux drivers are free.
 
-#### I want to buy a Chromebook to run Windows or Linux or macOS -- which should I buy?
-* Buying any Chromebook with the intention of running macOS, Windows or Linux is not a great idea. -- DO NOT EXPECT EVERYTHING TO WORK OUT OF THE BOX. Older models may fair better compatibility wise, but there are still lots of caveats, and it's not recommended to buy a Chromebook as a cheap Linux device.
+### Why is there a license error in Device Manager after installing audio drivers?
+* You messed up the license setup. Please refer to [CoolStar's driver portal](https://coolstar.org/chromebook/driverlicense/login.html).
 
-#### How can I get audio working under Linux?
-* Please see [here](https://github.com/WeirdTreeThing/chromebook-linux-audio)
+### Will the audio driver still work when re-installing Windows?
+* Yes, on the device the license was created. If the internal drive was replaced, see below.
 
-#### How do I get my top row keys on Linux to act like how they did on chromeOS?
-* Follow the instructions [here](https://github.com/WeirdTreeThing/cros-keyboard-map).
+### I upgraded my Chromebooks' internal drive and my audio/Thunderbolt 4 license is not working, what do I do?
+* The license is tied to the serial number of your internal drive. **You do not need to buy another license**. Send CoolStar a message through Patreon, and wait for about a week.
 
-#### I just want to boot Linux from USB on my Chromebook, what do I need to do?
-* Check the Supported Devices page, to ensure your device has functional RW_LEGACY firmware available. If so, flash RWL (RW_LEGACY firmware) and use that.
+### How do I buy Windows audio drivers in bulk?
+* A discount is only available for Chromebooks containing an `Intel 6th gen processor (up to Core i5)`, `N3350`, `N4000`, or, `N4020` processor, otherwise you must pay full price. You must purchase 10 or more licenses to be eligible for a discount. If you are eligible for a discount, you should ask in the [Chrultrabook Forum](https://forum.chrultrabook.com).
```

Comment:
This might need to be updated

Reply:
I dont recall Coolstar having announced new bulk discounts

Reply:
which models have bulk discounts is visible in the portal. No need to have duplicated and outdated info here in any case

Reply:
@ninelore where in the portal is this information?